.github/workflows/build.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   build:
+    if: false
```

Comment:
Is this supposed to still be here?

This action runs the lint tests on every PR which we should still be doing.

Reply:
I've disabled this workflow on purpose. It has been moved to _build_ job within introduced workflow. At the end of the day _build.yml_ will be removed.

Reply:
OK, makes sense